<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CHostWidget</class>
 <widget class="QWidget" name="CHostWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>498</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>COCRMHostWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="mLeftVBoxLayout">
     <item>
      <layout class="QVBoxLayout" name="mLeftMostVBoxLayout">
       <item>
        <layout class="QHBoxLayout" name="mControlsHBoxLayout">
         <item>
          <layout class="QVBoxLayout" name="mControlsVertLayout">
           <item>
            <widget class="QPushButton" name="mVideoButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>Start video</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mCapturePhotoButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>Take photo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mLoadSampleButton">
             <property name="text">
              <string>Load sample</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mGetDistanceButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>Get distance</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mFeaturesButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>Features</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="mVideoSpacerDelimeter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="mFeaturesBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Features</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QFormLayout" name="mFeaturesGridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="mVertAngleLabel">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Vertical angle</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="mVertAngleSpinBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="specialValueText">
               <string/>
              </property>
              <property name="prefix">
               <string/>
              </property>
              <property name="maximum">
               <number>180</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="mHorAngleLabel">
              <property name="text">
               <string>Horizontal angle</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="mHorAngleSpinBox">
              <property name="maximum">
               <number>180</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="mShiftDistance">
              <property name="text">
               <string>Shift distance</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="mShiftDistanceSpinBox">
              <property name="maximum">
               <number>500</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="mGalleryDelimiter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="mGraphicsLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>640</width>
       <height>480</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>mFeaturesButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFeaturesBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mVideoButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>mGraphicsLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
